### PR TITLE
hark: read all in group, graph

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
- Adds %read-graph, to dismiss all unreads and notifications for a graph.
- Adds %read-group, which calls %read-graph on all the graphs in a specified group
- Binds %read-group to `Shift+Escape`
- Adds a 'Mark all as Read' button to publish

This is intended to backstop and mitigate wonky unread counts due to note deletion. An actual fix is otw, but requires more testing as it's a mark upgrade. 
Screen Shot 2021-05-26 at 2 12 40 pm

### CODE REVIEW

**Comment by matildepark** [comment]
@liam-fitzgerald conflicts on hark-store

**Comment by liam-fitzgerald** [comment]
ping @tacryt-socryp review?

File: pkg/arvo/app/hark-store.hoon [hunk 1]
```diff
@@ -566,10 +570,53 @@
       (~(put by last-seen) stats-index new-time)
     (give %seen-index new-time stats-index)
   ::
+  ++  get-stats-indices
+    |=  rid=resource
+    %-  ~(gas ^in *(set stats-index:store))
```
Comment (tacryt-socryp) [inline pkg/arvo/app/hark-store.hoon:575]: Needing to constantly remember to do ^in is... unfortunate. I'd consider a different variable name so as not to shadow here.

**Review by tacryt-socryp — APPROVED** [review]
Sane. LGTM

File: pkg/arvo/app/hark-store.hoon (hunk 1 above)
Comment (liam-fitzgerald) [reply pkg/arvo/app/hark-store.hoon:575]: yeah ik, hopefully can get to a style refactor soonish

**Pull request merged**